### PR TITLE
termux-packages: Update android 7+ info. about upgrading

### PR DESCRIPTION
**Pull request by 0xXA**

### CODE REVIEW

**Comment by Grimler91** [comment]
Hi, thanks for the suggestions! Unfortunately `grep -P` isn't a supported option in busybox's version of grep, and not all users have installed the full grep package. 
We should probably keep the instructions as general as possible to try to make sure it works for as many users as possible

**Comment by 0xXA** [comment]
> Hi, thanks for the suggestions! Unfortunately `grep -P` isn't a supported option in busybox's version of grep, and not all users have installed the full grep package.
> We should probably keep the instructions as general as possible to try to make sure it works for as many users as possible

Fixed.
Preserved compatibility with busybox.

File: README.md [hunk 1]
```diff
@@ -32,11 +32,14 @@ Also, if you want to request a particular package, you may suggest it in an issu
 
 If your device running Android OS with version 7 and higher, it is highly recommended to check whether your Termux installation uses our new repository with packages compiled specially for higher Android API levels (24+).
 
-Execute command `termux-info` or look into file `$PREFIX/etc/apt/sources.list`. There should be a line containing following URL:
-```
-https://dl.bintray.com/termux/termux-packages-24/
+Execute any one of these commands:
+```bash
+$ termux-info | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3'
+
```
Comment (xeffyr) [inline README.md]: Commands should be **generic as possible**.

File: README.md [hunk 2]
```diff
@@ -32,11 +32,14 @@ Also, if you want to request a particular package, you may suggest it in an issu
...
+$ cat $PREFIX/etc/apt/sources.list | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3'
 ```
```
Comment (xeffyr) [inline README.md]: Same as above.
Why not to prompt user to download some shell script then ?

File: README.md [hunk 3]
```diff
@@ -32,11 +32,14 @@ Also, if you want to request a particular package, you may suggest it in an issu
...
-https://dl.bintray.com/termux/termux-packages-24/
+Execute any one of these commands:
+```bash
+$ termux-info | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3
+
+$ cat $PREFIX/etc/apt/sources.list | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3
 ```
-If not, then it is time to upgrade your installation. This procedure will involve complete erasing of `$PREFIX`, directory where all packages are installed (aka rootfs) but your home directory will be untouched.
+
+both of these commands produce output of `24`.If not, then it is time to upgrade your installation. This procedure will involve complete erasing of `$PREFIX`, directory where all packages are installed (aka rootfs) but your home directory will be untouched.
```
Comment (xeffyr) [inline README.md]: > both of these commands produce output of `24`.

If it produce output of "24", then **no upgrade needed**.

Understand the difference between `termux.net` and `https://dl.bintray.com/termux/termux-packages-24`. Also, do not rely on "24" at the end.

File: README.md (hunk 2 above)
Comment (0xXA) [reply README.md]: > Why not to prompt user to download some shell script then ?
Downloading and running is more cumbersome then copy-pasting

File: README.md (hunk 1 above)
Comment (0xXA) [reply README.md]: It's already fair enough then making user find some line of text in a bunch of lines contained in a file or from output of a command

Comment (xeffyr) [reply README.md]: Looks like you never tried to copy-paste **long** lines from Github.
And if line was meant to be copy-pasted, then why you included `$` at begin ?

**Pull request closed without merging**